src/comprehensions.py

```diff
@@ -11,13 +11,16 @@
 # Write a list comprehension to produce the array [1, 2, 3, 4, 5]
 
 y = []
-
+for a in range(5):
```

Comment:
Though you're basically doing the same thing with the loops, this file was asking to use list comprehensions. You can see some examples here https://www.programiz.com/python-programming/list-comprehension